bundle/src/main/java/com/adobe/acs/commons/properties/util/PropertyAggregatorUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2020 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.properties.util;
+
+import com.adobe.acs.commons.properties.PropertyConfigService;
+import com.day.cq.wcm.api.Page;
+import org.apache.sling.api.resource.ValueMap;
+
+import java.util.Map;
+import java.util.Set;
+
+public class PropertyAggregatorUtil {
+
+    private PropertyAggregatorUtil() {
+        throw new IllegalStateException("PropertyAggregatorUtil is a utility class, it should not be instantiated.");
```

Comment:
Looks like the standard in ACS AEM Commons for private ctors is to leave them completely empty.

Reply:
Codeclimate initially complained that I wasn't doing this, but it may be a 'minimum' of the private constructor, not necessarily having to throw this exception. I'll update this and see if codeclimate is okay with it 😄